src/situations/plan_de_la_ville/data/apps.js

```diff
@@ -53,8 +55,19 @@ const saisieBoulangerie = {
   choix: []
 };
 
+const deplacementDroiteMaisonVerte = {
+  id: 'deplacement-droite-maison-verte',
+  type: 'action',
+  nom_technique: 'deplacement_droite_maison_verte',
+  illustration: DeplacementDroiteMaisonVerte,
+  description: 'Dans quelle direction le personnage situé au milieu de l’écran doit-il aller pour se rendre à la maison verte ?',
+  intitule: 'Pour répondre, appuyez sur la flèche gauche ou droite de votre clavier.',
+  extensionVue: 'deplacement-droite-maison-verte',
+  choix: []
+};
+
 const configurationNormale = {
-  questions: [clicMaisonBleue, questionCouleur, saisieBoulangerie]
+  questions: [clicMaisonBleue, questionCouleur, saisieBoulangerie, deplacementDroiteMaisonVerte]
```

Comment:
Attention ici l'ordre des exercices n'est pas bon : 

1. Clic maison bleue
2.  Drag and Drop
3.  Boulangerie
4. Déplacement à droite
5.  Question couleur